### PR TITLE
Style version picker and extract methods

### PR DESCRIPTION
**Pull request by paulcsmith**
Hi Stephen!

While looking around I had some ideas for extracting methods on the page to make it easier to scan and make it easier to update the version picker. I also did some small styling tweaks.

Feel free to close or go a different direction though! 

Screen Shot 2020-06-22 at 11 37 08 AM

### CODE REVIEW

File: src/pages/versions/compare_page.cr [hunk 1]
```diff
@@ -5,27 +5,32 @@ class Versions::ComparePage < MainLayout
...
+  end
 
-      select_tag name: "to", class: "border rounded pr-2" do
-        Version::SUPPORTED_VERSIONS.each do |version|
-          option version, attrs: [version == to ? :selected : nil].compact
-        end
+  def version_picker(*, input_name, selected_version)
+    select_tag input_name: input_name, class: "border rounded pr-2" do
+      Version::SUPPORTED_VERSIONS.each do |version|
+        option version, attrs: [version == selected_version ? :selected : nil].compact
```
Comment (stephendolan) [inline src/pages/versions/compare_page.cr:20]: @paulcsmith Out of curiosity, is there a better (less verbose) way of tackling an `attr` that's shown or hidden based on a boolean value in Lucky?

My heart yearned to write something like [what Vue has](https://vuejs.org/v2/guide/syntax.html#Attributes), but I couldn't quite get something working that felt "right".

**Comment by stephendolan** [comment]
This is super clean, thank you for sharing!

File: src/pages/versions/compare_page.cr (hunk 1 above)
Comment (paulcsmith) [reply src/pages/versions/compare_page.cr:20]: There should definitely be an easier way. What we'll likely do is allow you to use `nil` in attributes array, but may also have some kind of helper that let's you do something like `attrs: {:selected => selected_version == version}`. But I think if we allow `nil` it'd be pretty smooth: `attrs: [:selected if version == selected_version]`

But maybe there is another approach we can use. I'm not sure exactly what it'd look like though 🤔Def open to ideas!

Comment (stephendolan) [reply src/pages/versions/compare_page.cr:20]: @paulcsmith opened up https://github.com/luckyframework/lucky/issues/1202 for discussion!

Comment (paulcsmith) [reply src/pages/versions/compare_page.cr:20]: Awesome! Thank you

**Pull request merged**